packages/conf-libgccjit/conf-libgccjit.12/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "2.0"
+maintainer: "Lukasz Stafiniak <lukstafi@gmail.com>"
+authors: [
+    "David Malcolm <dmalcolm@redhat.com>"
+]
+homepage: "https://gcc.gnu.org/wiki/JIT"
+license: "GPL-2.0-or-later"
+# Unfortunately, we cannot rely on the OS for selecting the right version
+# of libgccjit-dev, the generic dev package does not seem to exist on some
+# systems. There is libgccjit0, but not libgccjit0-dev (even though there is
+# libgccjit-devel on Fedora, libgccjit on NixOS, libgccjit-dev on Alpine).
+build: [
+  ["sh" "-exc" "gcc --version | grep 12"]
+  ["sh" "-exc" "echo \"#include  \\\"libgccjit.h\\\"\" > test.c"]
+  ["sh" "-exc" "gcc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos"}
+  [
+    "sh"
+    "-exc"
+    "cc -c $CFLAGS -I/opt/homebrew/include -I/opt/local/include -I/usr/local/include test.c"
+  ] {os = "macos"}
+]
+depends: [ "conf-gcc" ]
+# Check that the default version of gcc is not too high.
+available: [
```

Comment:
I am in favour of removing this and make the package fail in ci for those cases. Users can install the library manually, e.g. from sources, and use it also on those systems. Setting availability would prevent them from doing it